app/src/componentsV2/BottomSheet/components/BottomSheetWithoutContext/BottomSheetWithoutContext.tsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import { Col, Tabs, TabsProps } from "antd";
+import "../../BottomSheet.scss";
```

Comment:
Add comment/doc string somewhere that we had to create this separate component here; _in a reusable manner_; only to be able to use this stylesheet